packages/core/src/components/DropdownNew/__stories__/Dropdown.stories.tsx

```
@@ -424,7 +424,7 @@ export const DropdownShowSelectedItems: Story = {
           label="Label"
           required
           multi
-          showSelectedOptions
+          showSelectedOptions={false}
```

Why is it disabled? Curious

---

We had changed the default to be true, so the story is now "Hide selected items" and I forgot to update the example